app/sose2021/views.py

```diff
@@ -0,0 +1,310 @@
+
+# coding=utf-8
+from . import sommer21
+from flask import render_template, session, redirect, url_for, flash, current_app
+from flask_wtf import FlaskForm
+from wtforms import StringField, SelectField, SubmitField, BooleanField, validators, IntegerField
+from wtforms.fields.html5 import DateField
+from wtforms.widgets import TextArea
+from wtforms.widgets.html5 import NumberInput
+from app.oauth_client import oauth_remoteapp, getOAuthToken
+import json
+from datetime import datetime, time, timezone
+import pytz
+
+REGISTRATION_SOFT_CLOSE = datetime(2021, 5, 12, 21, 59, 59, tzinfo=pytz.utc)
+REGISTRATION_HARD_CLOSE = datetime(2021, 5, 12, 21, 59, 59, tzinfo=pytz.utc)
```

Comment:
REGISTRATION_HARD_CLOSE sollte nach REGISTRATION_SOFT_CLOSE sein, wenn irgendwie möglich

Reply:
REGISTRATION_HARD_CLOSE muss dann auch in /app/priorities/views.py:102 gesetzt werden